bears/c_languages/CPPCleanBear.py

```diff
@@ -19,6 +19,7 @@ class CPPCleanBear:
     AUTHORS = {'The coala developers'}
     AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
     LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Smell', 'Redundancy', 'Security'}
```

Comment:
i know this may fit into Redundancy, but i think ``Unused Code`` should be mentioned. I used it a lot. unused code is sometimes different from redundancy